docs/websites/nginx/nginx-ssl-and-tls-deployment-best-practices.md

```diff
@@ -0,0 +1,286 @@
+    :   ~~~
+        server_tokens       off;
+        ~~~
+
+2.  Save your changes and restart nginx.
+
+        systemctl restart nginx
+
+After restarting, direct your web browser to a directory of your server that does not exist, and nginx will no longer share its version number.
+
+[![404 With Server Tokens Disabled](/docs/assets/404_Not_Found_Server_Tokens_Off.jpg)](/docs/assets/404_Not_Found_Server_Tokens_Off.jpg) 
+
+## Enable HTTP/2 Support
+
+In September 2010, Google released the SPDY protocol for all versions of Chrome 6. SPDY is currently being phased out in favor of HTTP/2. Support for SPDY in Chrome was removed in May 2016. SPDY is supported only in nginx 1.8.x or older, whereas versions beginning with 1.9.5 are beginning to support HTTP/2. To check your nginx version:
+
+    nginx -v
+
+{: .note}
+> If you installed nginx from source without modifying your environment variables, invoke the full path to the binary:
+>
+>     /opt/nginx/sbin/nginx -v
+
+HTTP/2 is a new version of the HTTP standard replacing HTTP/1.1 to reduce page load time. Traditionally, when a user accessed a web page, a separate HTTP connection was established to load each resource (e.g. HTML, CSS, JavaScript, or images). HTTP/2 allows concurrent requests on a single connection to download assests in parallel. The server also compresses assets before sending them to the client, which requires less bandwdith. 
+
+{: .note}
+> Chrome has deprecated Next Protocol Negotiation (NPN) and now requires Application-Layer Protocol Negotiation (ALPN) for HTTP/2 compatibility. However, ALPN requires OpenSSL 1.0.2+. Many distributions, such as Debian 8 (Jessie) do not include this package in their repositories. If you intend to enable HTTP/2, you will need to use a version of nginx compiled with OpenSSL 1.0.2+. See our instructions on [compiling nginx from source](/docs/websites/nginx/install-nginx-and-a-startssl-certificate-on-debian-8-jessie#compile-nginx-from-source) for more information.
+
+1.  To enable HTTP/2, open your nginx SSL virtual host configuration file. Depending on how you installed nginx, this could be located at `/etc/nginx/sites-enabled/default` or at `/etc/nginx/conf.d/example_ssl.conf`. Look for the `listen` line within the "SSL Configuration" section. Uncomment the following line if necessary and add `http2` to the end before the semicolon.
+
+    {: .file-excerpt}
+    /etc/nginx/conf.d/example_ssl.conf
+    :   ~~~ conf
+        listen       443 ssl http2;
+        ~~~
+
+2.  Save your changes and restart nginx.
+
+        systemctl restart nginx
+
+3.  Open a web browser and navigate to [the KeyCDN HTTP/2 Test](https://tools.keycdn.com/http2-test), enter your Linode's domain name or hostname in the text box and click "Test". Optionally, uncheck the Public checkbox if you do not want your results displayed publicly. This free tool will check your server and let you know if HTTP/2 and ALPN are enabled and functioning correctly.
```

Comment:
Proof notes: "Test". = "Test."    
Also, Public checkbox = "Public" checkbox (This maintains consistency of checkbox identification)